server/container/guice/pom.xml

```diff
@@ -99,17 +99,17 @@
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-imap</artifactId>
+                <artifactId>james-server-jmap-rfc-8621</artifactId>
```

Comment:
Did you remove dependency management for `james-server-guice-imap`? I only see it removed...

Reply:
My bad.